src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,112 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static RegistrationService registrationService;
+    private static final int ADULT_AGE = 18;
+    private static final int BELOW_ADULT_AGE = 17;
+    private static final int ABOVE_ADULT_AGE = 19;
+    private static final String DEFAULT_VALID_LOGIN = "UserLogin";
+    private static final String DEFAULT_VALID_PASSWORD = "UserPassword";
+    private static final String SHORT_PASSWORD = "Short";
+    private User user;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @BeforeEach
+    void setUp() {
+        user = new User();
+        user.setAge(ABOVE_ADULT_AGE);
+        user.setPassword(DEFAULT_VALID_PASSWORD);
+        user.setLogin(DEFAULT_VALID_LOGIN);
+    }
+
+    @AfterEach
+    void tearDown() {
+        Storage.people.clear();
+    }
+
+    @Test
+    void registerUserIsNullThrowsException_OK() {
```

Comment:
```suggestion
    void registerUserIsNullThrowsException_notOk() {
```
the first letter after _ in lowercase (check it everywhere)
when the method expected exception is it ```notOk```))